.github/docker/Dockerfile.build

```diff
@@ -33,28 +33,33 @@ COPY --from=flatc-build /usr/local/lib/cmake/flatbuffers /usr/local/lib/cmake/fl
 COPY --from=flatc-build /usr/local/lib/pkgconfig/flatbuffers.pc /usr/local/lib/pkgconfig/flatbuffers.pc
 
 
+
 ENV USER=solana
 ARG SOLANA=1.18.22
+ARG RUST_VERSION=1.81.0
+RUN rustup toolchain install ${RUST_VERSION} && \
+    rustup default ${RUST_VERSION} 
 ENV CARGO_HOME=/usr/local/cargo
 ENV RUSTUP_HOME=/usr/local/rustup
 ENV PATH=${PATH}:/usr/local/cargo/bin:/go/bin:/home/${USER}/.local/share/solana/install/releases/${SOLANA}/bin
 
-# Set user and working directory
+# Set working directory
 ARG PACKAGE=bonsol
 WORKDIR /workspaces/${PACKAGE}
-USER solana
```

Comment:
Are you having issues running in user space?  it is a good practice because running as root can mask setup issues that would be encountered by real users